tests/Dockerfile

```diff
@@ -59,6 +59,12 @@ RUN curl -LOs "https://github.com/neilpa/yajsv/releases/download/v${YAJSV_VERSIO
     install -Tm 755 yajsv.linux.amd64 /usr/local/bin/yajsv && \
     rm yajsv.linux.amd64
 
+ENV DOCKER_VERSION "26.1.4"
+RUN curl -LOs "https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz" && \
+    tar -zxvf "docker-${DOCKER_VERSION}.tgz" docker/docker && \
+    install -Tm 755 docker/docker /usr/local/bin/docker && \
+    rm -rf docker "docker-${DOCKER_VERSION}.tgz"
```

Comment:
What unit tests depends on docker?

Reply:
Good catch! https://github.com/elastisys/compliantkubernetes-apps/pull/2157/commits/063cba237313a747cd5d264b7ef47b99c23630fd
But that begs the question, is there a unit test that requires kubelogin?

Reply:
That is an excellent question :sweat_smile: 
Non as far as I know in regards to the static ones. But the current check for the diagnostic script might need it, though it will eventually be moved to end-to-end instead.